### DISCUSSION TITLE
clearable on va-input doesn't work for me

### DISCUSSION THREAD

**Discussion by githubv5pw** [epicmaxco/vuestic-ui#804] [Q&A] [answered]
I'm on vuestic-admin 2.0.0 (v2.1.0) with vuestic-ui ~0.2.2 and the clearable prop mentioned in the docs doesn't seem to work for me on va-inputs. As far as I understand it should show some icon for clearing the input, but no icon is shown. I have va-selects, too, and for them a icon for clearing the input is shown. Am I missing something or is there a bug? Do you need any additional information? Thanks in advance.

**Comment by asvae** [comment, marked as answer]
@githubv5pw hey, sorry for late answer (had no notifications for discussions :X).

You'd need `vuestic-ui@1.0+` for these to work. `0.2.2` is pretty old version.

**Reply by githubv5pw** [reply]
Thanks for your reply. So I think it will be best to upgrade from vuestic-admin v2.1.0 to v3.0.0.
